bcipy/parameters/parameters.json

```diff
@@ -611,6 +611,14 @@
     "recommended_values": "",
     "type": "int"
   },
+  "max_incorrect": {
+    "value": "3",
```

Comment:
Let's set the default to five since that is the current norm for the clinical team